packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js

```diff
@@ -5,6 +5,8 @@
 /* eslint-disable object-curly-newline */
```

Comment:
@rana22 Can we avoid es-lint-disable? If possible can we remove some as well if those are quick fix.

Reply:
Removed